redis_opentracing/__init__.py

```diff
@@ -0,0 +1,149 @@
+from functools import wraps
+
+import redis
+
+g_tracer = None
+g_trace_prefix = None
+g_trace_all_classes = True
+
+def init_tracing(tracer, trace_all_classes=True, prefix='Redis'):
```

Comment:
Sure, the plan was to add it today at some point (with a good, clear README as well) :)